.travis.yml

```diff
@@ -13,7 +13,7 @@ script:
   - ruby ./scripts/image_checker.rb
   - docker run --volume=$(pwd):/app --workdir=/app coala/base:pre /bin/bash -c
     "coala --ci --apply-patches --no-orig;
-    coala-ci || true"
+    coala --ci"
```

Comment:
@jayvdb it's being run twice still unless line 16 is removed...I added `--ci` I will make this chnage -- back this change out @yukiisbored